swag/blockchain/blockchain.py

```diff
@@ -129,6 +132,10 @@ def swaggest(self):
         for key, _ in self.forbes:
             return key
 
+    @property
+    def yfu_nbr(self):
```

Comment:
Plz

"`yfu_nbr`"

xD

Maybe `next_yfu_id` would be better

Reply:
Lmao sorry I wasn't so inspired  your suggestion is 1000 time better

Reply:
Resolved in 023ca39